packages/ssm/README.md

```diff
@@ -57,7 +57,7 @@ npm install --save @middy/ssm
 - `cacheExpiry` (number) (default `-1`): How long fetch data responses should be cached for. `-1`: cache forever, `0`: never cache, `n`: cache for n ms.
 - `setToEnv` (boolean) (default `false`): Store role tokens to `process.env`. **Storing secrets in `process.env` is considered security bad practice**
 - `setToContext` (boolean) (default `false`): Store role tokes to `handler.context`.
-- `onChange` (function) (optional): Calls function when role tokens change after being initially set.
+- `onChange` (function(data)) (optional): Calls function when role tokens change after being initially set.
```

Comment:
`role token` doesn't make sense in this context

Reply:
maybe `parameter`?

Reply:
Good catch, I also need to make another tweak to ensure it doesn't call on init.